docs/Changelog-Platform.md

```diff
@@ -4,6 +4,30 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/)
 
+## <a name="1.7.1" />[1.7.1] - 2025-05-27
+
+See full log [of v3.9.0...v3.9.1](https://github.com/microsoft/testfx/compare/v3.9.0...v3.9.1)
+
+### Fixed
+
+* VSTestBridge: Handle TestPropertyAttributes.Trait instead of special casing specific properties by @youssef-backport-bot in [#5648](https://github.com/microsoft/testfx/pull/5648)
```

Comment:
We should be replacing any backport-bot with authors of the original PRs

Reply:
mkay, could you fix it though in the bot to do it for me?  e.g. like this? https://github.com/pester/Pester/blob/main/.github/workflows/backport.yml#L20

So I can then just delete the end of the PR name and add link?

Reply:
if your bot is based on the dotnet/runtime bot then you should sync with it, I had to do some small changes there to have all that data available.